### PR TITLE
homeWork 2.2

### PR DESCRIPTION
**Pull request by khOksana99**

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -13,4 +13,18 @@
...
+        Machine bulldozer = new Bulldozer();
+        bulldozer.doWork();
+        Machine excavator = new Excavator();
+        excavator.doWork();
+        track.stopWork();
+        bulldozer.stopWork();
+        excavator.stopWork();
+    }
 }
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/MainApp.java:29]: extra empty lines are redundant

File: src/main/java/core/basesyntax/Track.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    @Override
+    public void doWork() {
+        System.out.println("Track is working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Track stop working");
+    }
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Track.java:14]: an extra empty line is redundant

Comment (khOksana99) [reply src/main/java/core/basesyntax/Track.java]: done

File: src/main/java/core/basesyntax/MainApp.java (hunk 1 above)
Comment (khOksana99) [reply src/main/java/core/basesyntax/MainApp.java]: done

**Pull request closed without merging**